service/awsconfig/v5/resource/cloudformation/update.go

```diff
@@ -64,23 +65,27 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 		return cloudformation.CreateStackInput{}, microerror.Mask(err)
 	}
 
-	r.logger.LogCtx(ctx, "debug", "finding out if the main stack should be updated")
-
 	updateState := cloudformation.UpdateStackInput{}
 
-	if currentStackState.Name != "" && !reflect.DeepEqual(desiredStackState, currentStackState) {
-		r.logger.LogCtx(ctx, "debug", "main stack should be updated")
-
-		mainTemplate, err := r.getMainGuestTemplateBody(customObject)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-		updateState.StackName = aws.String(desiredStackState.Name)
-		updateState.TemplateBody = aws.String(mainTemplate)
-		// CAPABILITY_NAMED_IAM is required for updating IAM roles (worker policy).
-		updateState.Capabilities = []*string{
-			aws.String(namedIAMCapability),
+	if updateallowedcontext.IsUpdateAllowed(ctx) {
```

Comment:
I miss the bigger picture here. Why would we disable updates?

Reply:
+1, isn't it enough to have a version bundle version in WIP? AFAIK you can deploy WIP vbv with gsctl.

Reply:
And you have also the e2e tests, which exercise both latest WIP and active and where you can define exactly what to test.

Reply:
My understanding is we do this so updates are enabled in our test installations but disabled in production installations. IMO that is useful until we have full confidence in updating clusters.

It can also be used for change freezes in production installations. 

Is that correct? Some comments on this would be useful.

Reply:
The configuration is to have control over the feature because it is not yet complete and we will take some time to finish this feature. We need to test it. We need to fix bugs. We need to polish it. We need time to roll it out and meanwhile things happen left and right and out of a sudden we freeze a version because a customer needs some fix and then I have to sort disabling updates because it is just not ready. Now v5 resources are part of a WIP release. This might look like differently tomorrow. We should actually transition versions faster and should not keep them "open" for too long.